80_extension_and_integration/20_Rudder_plugins.txt

```diff
@@ -76,3 +76,44 @@ You also can interact with Rudder developers through the
 https://www.rudder-project.org/site/community/mailing-lists/[community] regular
 communication channels.
 
+
+==== Rudder Branding Plugin
+
+This plugin allows for changing the apparence of the login page, and the top bar of Rudder, by adding a colored bar with a text. Its main usage is to easily differentiate multiple Rudders server, and making sure the changes are made on the right environnement
+
+===== Prerequisite
+
+The rudder Branding plugin requires Rudder 4.3.3 or higher.
+
+===== Installing and Upgrading
+
+The installation and upgrade processes are exactly the same.
+Download the *rpkg* file, and run, on the Rudder server:
+
+----
+/opt/rudder/bin/rudder-pkg install-file rudder-plugin-branding-<Rudder version>-<plugin version>.rpkg        
+----
+
+It will add a default title bar and login bar, and a new page in Settings/Brandings.
+
+===== Default apparance
+
+Upon accessing Rudder, the login page will be changed, with a red line and text "Production", as shown here
+
+image:./images/branding/login-default.png[Default login message]
+
+Same for the Rudder top bar
+
+image:./images/branding/default-main-display.png[Default main display]
+
+====== Customization
+
+A new page is available within Rudder, to customize the apparence. This page is in the Settings section, called Branding. It allows for setting the top Bar and the Login page, with a color and a text, and a real-time display of the expected result
+
+image:./images/branding/Configure.png[Configure apparence]
+
+The text bar accepts any UTF-8 character or emojis, as displayed in this example
+
+image:./images/branding/Customised_bar.png[Customized bar]
```

Comment:
I must admit I checked if it was "Rudder 2000".

Reply:
:)

Reply:
![image](https://user-images.githubusercontent.com/76947/42167690-0ed6cca4-7e0f-11e8-972c-ea2993a386c0.png)